system/jlib/jtrace.cpp

```diff
@@ -224,13 +467,6 @@ class CSpan : public CInterfaceOf<ISpan>
         //Record the span as complete before we output the logging for the end of the span
         if (span != nullptr)
             span->End();
-
```

Comment:
I assumed you would keep it removed.  Please ask to clarify my meaning if it isn't clear or if you disagree. 

Before I saw the code removed I was expecting a separate PR to follow on short succession removing it. 
 Including the change in this PR meant I had to think through what other implications there would be - which I recorded in my comment.

I would remove them again - because adding them back raises complications about them not conforming to your json-only format for the mon evt logging.